dip-0006.md

```diff
@@ -140,11 +140,10 @@ Each phase lasts for multiple blocks (quorumDkgPhaseBlocks) to ensure that all m
 In this phase, the members of the new quorum are determined. This process is fully deterministic and results in exactly the same list seen by all members and observers.
 
 1. Retrieve the deterministic masternode list which is valid at quorumHeight
-2. Remove all entries from the list that are not confirmed on-chain in at least quorumMinMemberAge blocks
```

Comment:
This was not necessary? Or it is handled in another way?

Reply:
https://github.com/dashpay/dips/pull/42/commits/02c4b4a399a0b05482df4118177bafb125fdffc5